### PR TITLE
Feature/ap 15

### PR DESCRIPTION
**Pull request by zrshishir**
Parcel id validation with pending status.

### CODE REVIEW

File: internal/app/carrier/repository.go [hunk 1]
```diff
@@ -43,6 +45,15 @@ func (r *repository) InsertCarrierRequest(ctx context.Context, request model.Car
 }
 
 func (r *repository) UpdateCarrierRequest(ctx context.Context, parcel model.CarrierRequest, acceptStatus int, rejectStatus int, parcelStatus int, sourceTime time.Time) error {
+	var parcelValidation model.Parcel
+	if err := r.db.GetContext(ctx, &parcelValidation, validateParcelId, parcel.ParcelID, parcel.Status); err != nil {
```
Comment (rdhawladar) [inline internal/app/carrier/repository.go]: GetContext should use when you need need the row as output. As there is no use of parcelValidation you should use QueryContext.  But you don't need different query to know if parcel exist or not. You can check the output of updateAcceptQuery, if there is no rows effected you should commit and return with err msg.

Comment (zrshishir) [reply internal/app/carrier/repository.go]: QueryContext executes a query that returns rows, typically a SELECT. The args are for any placeholder parameters in the query.

Comment (rdhawladar) [reply internal/app/carrier/repository.go]: That's great. I think you didn't read the full comments as I mentioned no need to use different query. This is expensive.

**Review by rdhawladar — COMMENTED** [review]
Can't find the changes.

File: internal/app/carrier/repository.go (hunk 1 above)
Comment (zrshishir) [reply internal/app/carrier/repository.go]: That's right, I did not notice. I will test it and update it soon.

**Review by rdhawladar — APPROVED** [review]

**Pull request merged**